add-on/_locales/en/messages.json

```
@@ -726,5 +726,53 @@
   "page_landingWelcome_projects_title": {
     "message": "Related Projects",
     "description": "Projects section title (page_landingWelcome_projects_title)"
+  },
+  "option_header_telemetry": {
+    "message": "Telemetry",
+    "description": "A section header on the Preferences screen (option_header_telemetry)"
+  },
+  "option_telemetry_disclaimer": {
+    "message": "We're collecting minimal telemetry data to improve and prioritize our work. Please consent to the collection of these metrics to assist in our efforts!",
+    "description": "Disclaimer about telemetry collection in the telemetry section on the Preferences screen (option_telemetry_disclaimer)"
+  },
+  "option_telemetryGroupMinimal_title": {
+    "message": "Minimal metrics: session & companion views",
+    "description": "A title for the 'minimal' grouping of metrics we collect (option_telemetryGroupMinimal_title)"
+  },
+  "option_telemetryGroupMinimal_description": {
+    "message": "We send `session` and `view` events only if ipfs-companion is enabled and your browser is not idle.",
+    "description": "A description for the 'minimal' grouping of metrics we collect (option_telemetryGroupMinimal_description)"
+  },
+  "option_telemetryGroupMinimal_session_description": {
+    "message": "A `session` event - every 60 seconds.",
+    "description": "An explanation of the `session` telemetry we collect (option_telemetryGroupMinimal_session_description)"
+  },
+  "option_telemetryGroupMinimal_view_description": {
+    "message": "A `view` event - only for ipfs-companion pages.",
```

Users have no idea browser extensions are regular webpages behind the scenes.
Maybe more explicit:

```suggestion
    "message": "A `view` event for every time a part of extension's user interface is visible.",
```